cartodb/cartodb.py

```diff
@@ -138,7 +143,7 @@ def get_response_data(self, resp, parse_json=True):
                 return resp.json()
             return resp.content
         elif resp.status_code == requests.codes.bad_request:
-            raise CartoDBException(resp.json()['error'])
+            raise CartoDBException(resp.json()['errors'])
```

Comment:
Why have you changed this? 'error' is correct, if I'm not mistaken (you can run the tests to verify)